openfl-workspace/keras_nlp/src/nlp_dataloader.py

```diff
@@ -109,7 +116,9 @@ def _batch_generator(X1, X2, y, idxs, batch_size, num_batches):
             b = a + batch_size
             yield [X1[idxs[a:b]], X2[idxs[a:b]]], y[idxs[a:b]]
 
-    def _get_batch_generator(self, X1, X2, y, batch_size):
+    def _get_batch_generator(self, X1: np.ndarray, X2: np.ndarray,
+                             y: np.ndarray,
+                             batch_size: Optional[int]) -> Iterator[List[np.ndarray]]:
```

Comment:
```suggestion
                             batch_size: Optional[int] = None) -> Iterator[List[np.ndarray]]:
```

Reply:
We never use `batch_size` in `self._get_batch_generator(...)` as optional argument. So I don't think we should add `Optional[int] = None`, but we can get `None` as value, that's why I guess here should be `Optional[int]`